src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/PanelSummary/index.tsx

```diff
@@ -73,22 +69,9 @@ function PanelSummary({
     trackEvent("breakpoint.add_comment");
 
     if (pausedOnHit) {
-      createFrameComment(
-        currentTime,
-        executionPoint,
-        null,
-        { ...user, userId },
-        recordingId,
-        breakpoint
-      );
+      createFrameComment(currentTime, executionPoint, null, recordingId, breakpoint);
```

Comment:
I do think it would be nice to make the API for creating comments look more like:

`createComment(requiredParamsThatAllCommentsShare, OptionalCommentParams)`

Rather than all these different `createXComment` methods, but that's more of a future note, this PR is already addressing a ton of stuff.